src/main/java/uk/gov/service/notify/NotificationClient.java

```diff
@@ -23,6 +23,29 @@
     private final String baseUrl;
     private final Proxy proxy;
 
+    public NotificationClient(String apiKey, String baseUrl) {
+        this(
+                apiKey.substring(Math.max(0, apiKey.length() - 36)),
+                apiKey.substring(
+                        Math.max(0, apiKey.length() - 73),
+                        Math.max(0, apiKey.length() - 37)
+                ),
+                baseUrl
+        );
```

Comment:
note: i don't actually understand any of this crazy java ssl stuff so if it doesnt that's also cool